### PR TITLE
iosevka-comfy: 1.3.0 -> 1.4.0

### PR DESCRIPTION
**Pull request by DamienCassou**
## Description of changes

https://github.com/protesilaos/iosevka-comfy/releases/tag/1.4.0

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by DamienCassou** [comment]
Result of `nixpkgs-review pr 259054` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages failed to build:</summary>
  <ul>
    <li>iosevka-comfy.comfy</li>
    <li>iosevka-comfy.comfy-duo</li>
    <li>iosevka-comfy.comfy-fixed</li>
    <li>iosevka-comfy.comfy-motion</li>
    <li>iosevka-comfy.comfy-motion-duo</li>
    <li>iosevka-comfy.comfy-motion-fixed</li>
    <li>iosevka-comfy.comfy-wide</li>
    <li>iosevka-comfy.comfy-wide-duo</li>
    <li>iosevka-comfy.comfy-wide-fixed</li>
    <li>iosevka-comfy.comfy-wide-motion</li>
    <li>iosevka-comfy.comfy-wide-motion-duo</li>
    <li>iosevka-comfy.comfy-wide-motion-fixed</li>
  </ul>
</details>

**Comment by DamienCassou** [comment]
Result of `nixpkgs-review pr 259054` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li>iosevka-comfy.comfy</li>
    <li>iosevka-comfy.comfy-duo</li>
    <li>iosevka-comfy.comfy-fixed</li>
    <li>iosevka-comfy.comfy-motion</li>
    <li>iosevka-comfy.comfy-motion-duo</li>
    <li>iosevka-comfy.comfy-motion-fixed</li>
    <li>iosevka-comfy.comfy-wide</li>
    <li>iosevka-comfy.comfy-wide-duo</li>
    <li>iosevka-comfy.comfy-wide-fixed</li>
    <li>iosevka-comfy.comfy-wide-motion</li>
    <li>iosevka-comfy.comfy-wide-motion-duo</li>
    <li>iosevka-comfy.comfy-wide-motion-fixed</li>
  </ul>
</details>

**Review by nagy — CHANGES_REQUESTED** [review]
Have not tested it yet.

File: pkgs/data/fonts/iosevka/comfy.nix [hunk 1]
```diff
@@ -20,17 +20,17 @@ let
     "comfy-wide-motion-fixed" # Slab   | Wide    | Monospaced | No        |
     "comfy-wide-motion-duo"   # Slab   | Wide    | Duospaced  | Yes       |
   ];
-  version = "1.3.0";
-  src = fetchFromSourcehut {
+  version = "1.4.0";
+  src = fetchFromGitHub {
     owner = "~protesilaos";
```
[the remark this answers is not in the document]
Comment (DamienCassou) [reply pkgs/data/fonts/iosevka/comfy.nix]: Nice catch! Surprisingly this works anyway. One can download https://github.com/~protesilaos/iosevka-comfy/archive/1.4.0.tar.gz just fine (removing the `~` also works of course).

**Review by nagy — APPROVED** [review]
Tested and it works

**Comment by DamienCassou** [comment]
Thanks for testing and reporting back.

**Pull request merged**